src-java/testing/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/links/BfdSpec.groovy

```diff
@@ -71,6 +79,25 @@ class BfdSpec extends HealthCheckSpecification {
             }
         }
 
+        and: "BFD logical ports are created"
+        def srcSwLogPort = isl.srcPort + bfdLogPortDelta
+        def dstSwLogPort = isl.dstPort + bfdLogPortDelta
+        def srcSwitchAddress = northbound.getSwitch(isl.srcSwitch.dpId).address
+        def dstSwitchAddress = northbound.getSwitch(isl.dstSwitch.dpId).address
+        [isl.srcSwitch, isl.dstSwitch].each { sw ->
+            def swAddress = sw.dpId == isl.srcSwitch.dpId ? srcSwitchAddress : dstSwitchAddress
+            def swPort = sw.dpId == isl.srcSwitch.dpId ? isl.srcPort : isl.dstPort
+            def swLogPort = sw.dpId == isl.srcSwitch.dpId ? srcSwLogPort : dstSwLogPort
```

Comment:
`def swLogPort = swPort + bfdLogPortDelta` and remove L83-84

Reply:
fixed